client/src/invoker/grpc/account.rs

```diff
@@ -0,0 +1,144 @@
+use crate::invoker::grpc::proto;
+use crate::invoker::grpc::proto::bank_client::BankClient;
+use tonic::transport::Channel;
+use tonic::Response;
+
+/// Структура объекта вызова для работы со счетами.
+pub struct AccountInvoke {
+    client: BankClient<Channel>,
+}
+
+impl AccountInvoke {
+    /// Конструктор.
+    pub fn new(client: BankClient<Channel>) -> Self {
+        Self { client }
+    }
+}
+
+/// Интерфейс работы со счетами.
+#[trait_variant::make(IntFactory: Send)]
```

Comment:
Это копипаста из документации. Тут лучше переписать на:

```suggestion
#[trait_variant::make(Send)]
```